modules/nf-core/fastk/fastk/environment.yml

```diff
@@ -3,3 +3,5 @@ channels:
   - conda-forge
   - bioconda
   - defaults
+dependencies:
```

Comment:
Have things changed. I thought tools without conda could have this file removed?

Reply:
It fails nf-core/lint if this line and the next line are not added.

Reply:
That I understood. I meant for other modules this environment file was completely deleted.

Reply:
Let me try that!